third_party/binary/BUILD

```diff
@@ -20,8 +20,8 @@ remote_file(
 
 remote_file(
     name = "plz",
-    url = f"https://github.com/thought-machine/please/releases/download/v17.3.1/please_17.3.1_{CONFIG.OS}_{CONFIG.ARCH}",
     binary = True,
     test_only = True,
+    url = f"https://github.com/thought-machine/please/releases/download/v17.3.1/please_17.3.1_{CONFIG.OS}_{CONFIG.ARCH}",
```

Comment:
Could we set a variable here for the plz version to make this (very slightly) easier to update?